packages/playground/blueprints/src/lib/steps/login.spec.ts

```diff
@@ -25,10 +25,29 @@ describe('Blueprint step installPlugin', () => {
 	});
 
 	it('should log the user in', async () => {
+		await login(php, {
+			username: 'admin',
+		});
+		const result = await php.run({
+			code: `
+				<?php
+				echo PLAYGROUND_AUTO_LOGIN;
+			`,
+		});
+		expect(result.text).toContain('admin');
+	});
+
+	it('should redirect user to wp-login.php with correct redirect_to', async () => {
```

Comment:
@brandonpayton I kept only the first part of your test. This combined with a E2E test should be good enough for now.